api/descriptions/policies/compliance_ci_serverless_put.md

```diff
@@ -0,0 +1,53 @@
+Updates the compliance policy for serverless functions built in your Continuous Integration (CI) pipeline>.
+All rules in the policy are updated in a single shot.
+
+The policy set in this endpoint is enforced by the scanners in the Jenkins plugin and the `twistcli` command line tool.
+
+This endpoint maps to the policy table in **Defend > Compliance > Functions > CI** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+To construct an effective rule for this policy, specify at least one "check" and one `effect` value. 
+See [How to Construct a Compliance Policy](#how-to-construct-a-compliance-policy) for more info.
+
+For a full list of checks, go to **Defend > Compliance > Functions > CI** in the Console UI and create a new rule.
+All prebuilt checks and their IDs are shown under **Compliance actions**.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/compliance/ci/serverless' \
```

Comment:
@KICKdesigns This command isn't working for me. Please debug and revalidate.

BTW, the Console logs are a good way to get some feedback for non-working commands. You can use the search filter box (e.g. "compliance") to narrow the scope. It would be nice to mention this resource somewhere in the API docs.


Here's the error for this curl command:

<img width="1370" alt="curl-cmd-error-details" src="https://user-images.githubusercontent.com/6518946/105937513-52889b80-601b-11eb-8f6d-49090b164c2b.png">

Reply:
@iansk WIth the current push, the curl is working for me. Let me know if its still not working on your end.

Reply:
@KICKdesigns It's not working for me. The comma at the end of the second last line causes it to fail:

`"policyType": "ciServerlessCompliance",`

Please implement a QA process for all curl commands as follows:

1. Update the markdown file.
2. Build the API docs
3. Copy the curl command into a text file.
4. Fill in the `<VARS>`
5. Paste it into a shell window and run the command.
6. Go to the UI to validate it's working.

Customers continually complain about non-working curl commands, so I'd like our curl commands to be flawless.

Reply:
@KICKdesigns I fixed the curl command for you.